backend/routes/web.js

```diff
@@ -52,3 +52,5 @@ router.delete(
 
 router.patch("/recipe/share/:recipeId", authenticateToken, RecipeController.updateShareCount);
 export default router;
+
+router.get('/recipe-of-the-day', getRecipeOfTheDay);
```

Comment:
@sanchitc05 the way the function is added to the route is wrong please refer other routes and always put everything before the export in all changed files

Reply:
Okay @AlfiyaSiddique